polaris.shopify.com/src/components/IconsPage/icon-renames.ts

```diff
@@ -0,0 +1,1021 @@
+export const iconRenames = {
```

Comment:
Maybe name this file `icon-renames-v7-v8` and the export `iconRenamesV7V8` so it's versioned and we know when it got added and why quickly.